### DISCUSSION TITLE
Is it possible to import `Failed` properly?

### DISCUSSION THREAD

**Discussion by eliegoudout** [pytest-dev/pytest#13093] [Q&A] [answered]
Hello,

In my testing setup, I factored some tests in a "check_this_should_fail" that contains a few `pytest.raises` statements. Now, I'd like to do
```python
special_broken_behaviour = ... # created in this test
with pytest.raises(Failed):
    check_this_should_fail(special_broken_behaviour)  # Will not fail since specially broken
```
No, for this, I can use
```python
from _pytest.outcomes import Failed
```
but it's obviously not intended.

Is it on purpose to avoid some kind of anti-pattern? Are there any other way or should I drop this idea altogether?

I don't think it's trivially an anti-pattern to wanna make `pytest.raises` fail sometimes.

Thanks!

**Comment by RonnyPfannschmidt** [comment, marked as answer]
`pytest.fail.Exception`

**Reply by eliegoudout** [reply]
Aaaah thanks!
